packages/fields/package.json

```diff
@@ -35,6 +35,7 @@
 		"@babel/runtime": "7.25.7",
 		"@wordpress/api-fetch": "*",
 		"@wordpress/blob": "*",
+		"@wordpress/block-editor": "*",
```

Comment:
Why is this needed?

Reply:
Fixed with e0f518e4912a0cd7ebaf2af46e8e93cc2ac29da5.

Reply:
Is this commit not in the branch? For some reason GitHub is showing me the dependency addition still in the diff

Reply:
It should be okay now!